datumaro/plugins/coco_format/extractor.py

```diff
@@ -152,16 +154,22 @@ def _load_label_categories(self, json_cat, *, keep_original_ids):
         self._label_map = label_map
 
     def _load_person_kp_categories(self, json_cat):
-        categories = PointsCategories()
+        point_categories = PointsCategories()
+        label_categories = self._categories[AnnotationType.label]
+
         for cat in json_cat:
             label_id = self._label_map[self._parse_field(cat, "id", int)]
-            categories.add(
+            labels = self._parse_field(cat, "keypoints", list)
+            point_categories.add(
                 label_id,
-                labels=self._parse_field(cat, "keypoints", list),
+                labels=labels,
                 joints=self._parse_field(cat, "skeleton", list),
             )
 
-        self._categories[AnnotationType.points] = categories
+            for label in labels:
+                label_categories.add(label, cat.get("name"))
```

Comment:
How I should to write my annotation file If I want to have Skeleton with three points: head and two legs?
I have tried this:
```json
"categories":[{
        "id":1,
        "name":"a",
        "supercategory":"",
        "keypoints":[
          "leg",
          "head",
          "leg"
        ],
        "skeleton":[
          [0, 1],
          [1, 2]
        ]
}]
"annotations": [{
    "id": 1,
    "category_id": 1,
    "bbox": ...,
    "keypoints": [10, 20, 2, 20, 30, 2, 20, 10, 2]
}]
```
But got the error.

Reply:
Keypoint names can't match